pyqir-jit/src/jit.rs

```diff
@@ -94,6 +106,7 @@ fn choose_entry_point<'ctx>(
     }
 }
 
+#[allow(clippy::trivially_copy_pass_by_ref)]
```

Comment:
I didn't realize `FunctionValue` has `Copy`. You can remove the reference here.

Reply:
Changing the ref to `FunctionValue` makes the code above it no longer work as the calls expect `&FunctionValue`.

Reply:
You're right. You could change the `filter` call to a lambda that dereferences it, but I'm fine either way.